sdk/firstload/package.json

```diff
@@ -70,7 +70,10 @@
 	},
 	"size-limit": [
 		{
-			"path": "dist/**.js",
+			"path": [
+				"dist/*.js",
+				"!dist/*.umd.js"
+			],
```

Comment:
@Vadman97 it looked like the UMD bundle was including all necessary code to run Highlight and doesn't benefit from dynamic imports. I didn't see a way to trim it down any further, so decided to exclude it from this check. Do you think that is reasonable?

Reply:
Sounds good! FWIW, I don't think the main bundle size changed much. You can see the new chunk that will be conditionally loaded, but the other files are around the same size.

**Before**
<img width="552" alt="Screenshot 2024-06-25 at 5 13 03 PM" src="https://github.com/highlight/highlight/assets/308182/fa71087f-0018-4edb-a669-104991e02165">

**After**
<img width="518" alt="Screenshot 2024-06-25 at 5 14 23 PM" src="https://github.com/highlight/highlight/assets/308182/db97061d-85d0-4cf9-a3a2-f844f2dbd9f0">

Reply:
neat! yeah post gzip not a huge network difference